src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,12 @@
+package pages;
+
+import org.openqa.selenium.By;
+
+public class LoginPage{
+
+    public static final String URL = "https://192.168.100.26/ ";
+    public By email = new By.ByCssSelector("#Username");
+    public By password = new By.ByCssSelector("#Password");
+    public By submit = new By.ByCssSelector("#SubmitButton");
```

Comment:
Optional: I prefer to use static methods from By class, f.e. By.xpath(), By.cssSelector, By.id, etc.

Reply:
done